.pre-commit-config.yaml

```diff
@@ -49,25 +49,25 @@ repos:
     rev: "v0.942"
     hooks:
       - id: mypy
-        args:
-          - --show-error-codes
-          - --ignore-missing-imports
-        files: ^(src/|packages/.*/test|conftest.py|docs/)
+        args: [--ignore-missing-imports]
```

Comment:
This is really hard to disable, since every test file will have different imports based on what it's testing. But I did move the `/src` checking (after enabling it) over to the next check where this is not enabled.

Thinking about it, maybe `/src` can move there too, and thsi could just be the packages's non-src files and that's all.